packages/vulcan-core/lib/modules/containers/withMulti.js

```diff
@@ -46,6 +46,8 @@ import {
 } from 'meteor/vulcan:lib';
 import compose from 'recompose/compose';
 import withState from 'recompose/withState';
+import qs from 'qs';
+import { LocationContext, NavigationContext } from '../components/App'
```

Comment:
This makes me sad, but `routeUtil` isn't accessible from `vulcan-core` so I guess we have to do it for now.